fleetctl/fleetctl.go

```diff
@@ -745,6 +782,56 @@ func setTargetStateOfUnits(units []string, state job.JobState) ([]*schema.Unit,
 	return triggered, nil
 }
 
+// getBlockAttempts gets the correct value of how many attempts to try
+// before giving up on an operation.
+// It returns a negative value which means try forever, if zero is
+// returned then do not make any attempt, and if a positive value is
+// returned then try up to that value
+func getBlockAttempts() int {
+	// By default we wait forever
+	var attempts int = -1
```

Comment:
Yes I set it to negative value to not set it to zero, this way I can return immediately here https://github.com/endocode/fleet/commit/75b60886e6f698845db93e1cd9db89e0ebcb4398#diff-197bb752bf0f52567eb6b0023ea65e40R782

And lalso to allow us to continue with the previous logic inside "waitForUnitStates()" 0 or  any negative value. I didn't touch "waitForUnitStates()" since it's used in some other parts of the code, so I only converted new calls keeping the same semantics from a user experience. Yes the problem is we have two values inside the code blockattempts and noblock that interfere which other that's why I think we should convert this into one value that allows us to be more consistent in each part of the code. Perhaps later other parts can be converted too, but now I only changed fleetctl commands.